### PR TITLE
fixed touchmove errors

### PR DESCRIPTION
**Pull request by narasab**
fixed errors -> [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false'

### CODE REVIEW

File: packages/tiny-swiper/src/core/sensor/index.ts [hunk 1]
```diff
@@ -71,7 +71,9 @@ export function Sensor (
         if (options.touchMoveStopPropagation) e.stopPropagation()
 
         move(getPosition(e))
-        state.isTouching && e.preventDefault()
+        if(state.isTouching && e.cancelable){
```
Comment (joe223) [inline packages/tiny-swiper/src/core/sensor/index.ts:74]: `cancelable` property of `Event` maybe `undefined`

File: packages/tiny-swiper/src/core/sensor/index.ts [hunk 2]
```diff
@@ -88,7 +90,7 @@ export function Sensor (
                 passive: options.passiveListeners,
                 capture: false
             })
-            attachListener($el, 'touchmove', onTouchMove)
+            attachListener($el, 'touchmove', onTouchMove, { passive:true })
```
Comment (joe223) [inline packages/tiny-swiper/src/core/sensor/index.ts:93]: We should prevent page scroll while dragging slider item. But unable to preventDefault inside passive event listener invocation if we set `passive:true`, `e.preventDefault` will never be called at the same time.

**Review by joe223 — COMMENTED** [review]
@narasab Really appreciate the pull request 😃

Here are some suggestions:

- `e.preventDefault` will never be called if setting `passive: true`
-  we should prevent page scrolling while trying to drag sliders
- `Event.cancelable` maybe `undefined`

**Comment by narasab** [comment]
Thanks for the feedback. Here is a good summary I guess https://developers.google.com/web/updates/2017/01/scrolling-intervention.

**Comment by joe223** [comment]
Hi @narasab , default behavior should be prevented otherwise page scrolling will coexist with Swiper slide action. Or we have to tackle this problem with different method.

This video shows the concern as mentioned above.

https://user-images.githubusercontent.com/10026019/105037649-4049b300-5a99-11eb-93ea-a5ed3f2abdcc.mp4

checkout this example `packages/tiny-swiper/demo/vertical.html`

**Comment by joe223** [comment]
> css scroll snap with intersection observer

@narasab That‘s a great idea! You mean create slide effect with scroll snap property sets just like [this](https://codepen.io/michellebarker/full/PowYKXJ) right?

But what I'm concerned is

- CSS scroll snap is not as powerful as we want, we can't implement some effects only with CSS
- The compatibility of CSS scroll snap and JS intersection observer, Tiny-Swiper is trying to work with IE10+ for the next two years.

**Comment by narasab** [comment]
Yes like that one, and use intersection observer to emit the index. But you're right that has to wait for sometime if IE is a concern.

**Comment by joe223** [comment]
But we still can fix the touchmove errors,  right? Just make sure that e.cancelable is not `false`

```javascript
if(state.isTouching && e.cancelable !== false)
```

**Comment by narasab** [comment]
Yes you're right. Checked with the web view and all seems ok.

**Pull request merged**